subprojects/common/report-viewer/src/test/kotlin/com/avito/report/ReportsApiTest.kt

```diff
@@ -55,7 +62,9 @@ internal class ReportsApiTest {
 
     @Test
     fun `getReport - returns Error - when throws exception with no data`() {
-        mockWebServer.enqueue(MockResponse().setResponseCode(500))
+        repeat(retryCount) {
```

Comment:
Is it an imitation of "unrecoverable" case?
In this case we duplicate implementation here. We can show the intention by Dispatcher and don't rely on count.